dlk/backends/include/cpp.h

```diff
@@ -52,27 +52,6 @@ void conv(T_in in_data[], T_out out_data[], T_k k_data[], T_out threshold_data[]
 }
 /// @}
 
-/// convolution with kn2row and tiling
-/// @{
-void conv_kn2row_tiling_impl(T_in in_data[], T_out out_data[], T_k k_data[], T_out threshold_data[],
-                             const unsigned in_w, const unsigned in_h, const unsigned in_c, const unsigned out_w,
-                             const unsigned out_h, const unsigned out_c, const unsigned k_w, const unsigned k_h,
-                             const unsigned pad, const unsigned stride);
-
-template <int KH, int KW>
-void conv_kn2row_tiling(T_in in_data[], T_out out_data[], T_k k_data[], T_out threshold_data[], const unsigned in_w,
-                        const unsigned in_h, const unsigned in_c, const unsigned out_w, const unsigned out_h,
-                        const unsigned out_c, const unsigned pad, const unsigned stride)
-{
-  assert(((KH == 3) && (KW == 3)) || ((KH == 1) && (KW == 1)));
-  assert(((KH == 3) && (pad == 1)) || ((KH == 1) && (pad == 0)));
-  assert(stride == 1);
-
-  cpp::conv_kn2row_tiling_impl(in_data, out_data, k_data, threshold_data, in_w, in_h, in_c, out_w, out_h, out_c, KW, KW,
-                               pad, stride);
-}
-/// @}
-
 /// 2A 1W quantized convolution with 1x1 and 3x3 kernel
 template <int KH, int KW>
 void qconv(T_q in_data_packed[], T_out out_data[], T_q k_data_packed[], unsigned in_w, unsigned in_h,
```

Comment:
do we need qconv? :thinking: 

if it is something like implementation example we could probably separate it from the main one.